tests/e2e/cogs/dice_test.py

```diff
@@ -118,11 +118,11 @@ async def test_inline_rolling_reaction(avrae, dhttp, mock_ldclient):
     async with server_settings(avrae, inline_enabled=InlineRollingType.REACTION):
         with mock_ldclient.flags({"cog.dice.inline_rolling.enabled": True}):
             avrae.message("[[1d20]]")
-            await dhttp.receive_reaction("\N{game die}")
+            await dhttp.receive_reaction("\N{GAME DIE}")
```

Comment:
is this `game die` change (and the one that follows on line 125) actually case-sensitive?